src/core/unittest/SpinFrame.cpp

```diff
@@ -47,7 +47,11 @@ TEST(SpinFrame, SpinFrame1000000)
     BOOLEAN InvalidFrame;
     uint8_t Buffer[255];
     uint8_t BufferLength = 0;
+
     uint8_t FrameType;
+    CXPLAT_STATIC_ASSERT(
+        QUIC_FRAME_MAX_SUPPORTED <= (uint64_t)UINT8_MAX,
+        "Tests below assumes frames fit in 8-bits");
```

Comment:
Can you add an AckFrequency frame case to this test?

Reply:
Also, add a case in the QUIC_FRAME_TYPE ostream overload to pretty print the Ack Frequency frame

Reply:
We already have a test case for the frame.